admin/config.js

```diff
@@ -53,12 +53,13 @@ module.exports = {
     }
   },
   Logging: {
+    // @TODO: Remove hardcoded key
     LogDna: {
-      key: process.env.LOGDNA_API_KEY,
+      key: '5a3d75a421e412afe3139e4a02a6ce19', // process.env.LOGDNA_API_KEY,
```

Comment:
do not check in your LogDNA api key.  This is a secret.

You will need to re-generate it now